package.json

```diff
@@ -25,7 +25,7 @@
     "zone.js": "~0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular-devkit/build-angular": "^0.803.23",
```

Comment:
On my system this change is breaking the build as it's not compatible with @angular/cli 7.3.8

Reply:
./node_modules/.bin/ng version
Angular CLI: 7.3.9
Node: 12.14.1
OS: linux x64
Angular: 7.2.14
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.9
@angular-devkit/build-angular     0.803.23
@angular-devkit/build-optimizer   0.803.23
@angular-devkit/build-webpack     0.803.23
@angular-devkit/core              7.3.9
@angular-devkit/schematics        7.3.9
@angular/cli                      7.3.9
@ngtools/webpack                  8.3.23
@schematics/angular               7.3.9
@schematics/update                0.13.9
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.39.2

Reply:
> Task :buildWebClient

> container-jfr-web@0.0.0 build /home/jkang/Work/code/src/github.com/rh-jmc-team/container-jfr/web-client
> ng build

Schema validation failed with the following errors:
  Data path ".builders['app-shell']" should have required property 'class'.
Error: Schema validation failed with the following errors:
  Data path ".builders['app-shell']" should have required property 'class'.
    at MergeMapSubscriber.project (/home/jkang/Work/code/src/github.com/rh-jmc-team/container-jfr/web-client/node_modules/@angular-devkit/core/src/workspace/workspace.js:215:42)

Reply:
Ah, I see that now too after a `rm -rf dist node_modules; npm install; ng build`. Hmm.

Reply:
Curiously I cannot build with the package-lock.json in web-client commit e0981bd292fddf0c71b05b4ad0d9096c2baf3fbb

After deleting it and running npm install to generate a new one, npm audit reports 1 low, 3 moderate and 2 high. So that seems like an improvement.

For what it's worth, I have
npm 6.13.4
node v12.14.1

Reply:
I have npm 6.12.1 and node v12.13.1, which seem to be the latest versions for F31. One minor version point apart probably isn't our issue though.

Reply:
Do you see a significant difference in regenerating the package-lock.json on your system? Say in the results of npm audit? Maybe we can just do that for now and look at updating to angular-cli 8.x in a future issue.

Reply:
I was just working on trying to get through an Angular 8 upgrade, but it looks like that's going to be blocked currently by patternfly-ng only supporting <8.0.0, as well as a lower typescript version than the newer Angular stuff needs.

```
$ git co master
$ rm -rf node_modules
$ npm ci
... snip ...

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.11.0/linux-x64-72_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.11.0/linux-x64-72_binding.node": 

HTTP error 404 Not Found
```

Apparently this comes from using node 12+? https://github.com/sass/node-sass/issues/2694

What did you do that you're able to build now? It seems that my container-jfr submodule copy of -web is only working because of npm's weak, incremental dependency management, so some old cached copies are keeping it alive...

Reply:
I ran
```
$ rm -rf node_modules dist
$ rm package-lock.json
$ npm install
```

It gets node-sass 4.12.0 for me

Reply:
Ah okay, that makes sense. Doing that I now see the same 1 low/3 moderate/2 high result from npm audit, too.